deployments/examples/ocis_individual_services/docker-compose.yml

```diff
@@ -630,105 +786,85 @@ services:
       - groups
       - server
     environment:
-      OCIS_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
-      OCIS_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
-      OCIS_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
+      GROUPS_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
+      GROUPS_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
+      GROUPS_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
 
-      STORAGE_GROUPPROVIDER_ENDPOINT: groups:9160
-      STORAGE_GROUPPROVIDER_ADDR: 0.0.0.0:9160
+      GROUPS_GRPC_ADDR: 0.0.0.0:9160
 
-      STORAGE_GROUPPROVIDER_DRIVER: ldap
-      STORAGE_LDAP_URI: "ldaps://glauth:9126"
-      STORAGE_LDAP_INSECURE: "true"
-      STORAGE_LDAP_BIND_PASSWORD: ${STORAGE_LDAP_BIND_PASSWORD:-reva}
-      STORAGE_LDAP_IDP: https://${OCIS_DOMAIN:-ocis.owncloud.test}
+      GROUPS_LDAP_URI: ldaps://idm:9235
+      GROUPS_LDAP_CACERT: ""
+      GROUPS_LDAP_INSECURE: ${INSECURE:-true}
```

Comment:
we could omit the insecure option when sharing the idm cacert with volume to all services. Don't know how volumes exactly behave on docker swarm (or other really distributed docker deployments) but we could do this since we already need multi container/worker-node volume mounts for scaling services.